src/silo_api/api.cpp

```diff
@@ -77,7 +95,7 @@ class SiloServer : public Poco::Util::ServerApplication {
          handleApi();
       }
 
-      if (config().hasProperty("processData")) {
+      if (config().hasProperty("preprocessing")) {
          handleProcessData();
```

Comment:
How about returning directly after this? Or maybe even let `handleApi` and `handleProcessData` return something, so we could write `return handleProcessData()` here?